scripts/dapps.js

```diff
@@ -51,26 +56,27 @@ async function initialize() {
       { name: 'dapps', value: 'dapps', checked: true },
       { name: 'evan-libs', value: 'evan-libs', checked: false },
       { name: 'libs', value: 'libs', checked: false },
-    ]
+      { name: 'playground', value: 'playground', checked: false },
+    ],
   }])).dappCategories;
-  dappDirs = [ ]
+  dappDirs = []
     .concat(
```

Comment:
what is the reason for concat with an empty array? Isn't it just the same as setting the var directly?

`dappDirs = categories.map((key) => getDirectories(path.resolve(`../${key}`)))`